src/js/modifiers.js

```diff
@@ -275,23 +275,24 @@ function modify_rotate() {
   console.log(transposer)
   console.log(equave)
```

Comment:
I know the PR isn't about these lines, but if possible and no longer needed, then please remove these console logs